### ISSUE TITLE
Site not working

### ISSUE THREAD

**Issue by nombrekeff** [alexfernandez/package-quality#61]
Hi there, it seems the site at https://packagequality.com is not working. At least today.

**Comment by nombrekeff** [comment]
Is the project/site still maintained? 
If so I will suggest adding a message to the README.md

I'm willing to do it myself, and maybe maintain it, for a while at least.

**Comment by jcubic** [comment]
It's still not working, what happen to the project?

**Comment by alexfernandez** [comment]
Hi, the project is not maintained at the moment. The server stops randomly and I just reboot it. To be honest I don't even have access to the server any more. If you would like to maintain it go ahead! Just set the project up somewhere and I will check it. Thanks!

**Comment by nombrekeff** [comment]
Maybe I could maintain it, I was thinking of publishing it on zeit.now or similar, although domain would change. It would no longer be packagequality.com

**Comment by jcubic** [comment]
Maybe you can give the domain it's usualy in different place then the server. It would be great if domain state the same, I think that lot of projects use it and don't even know that's broken if that happen.

**Comment by nombrekeff** [comment]
Yup, agreed, maybe he could trespass the domain, and set it up in zeit.now.
Are you up to help me out? I don't have much time to do maintain it

**Comment by jcubic** [comment]
Sorry, I don't have time, I have few open source projects I'm maintaining and I'm working on those when I have time. No more time for new projects.

**Comment by nombrekeff** [comment]
Totally understandable, I'm in the same situation, I work full time and maintain a couple of projects as well.

**Comment by bennyn** [comment]
I received a "Failed to load resource: the server responded with a status of 504 (Gateway Time-out)" when querying through the site: https://packagequality.com/#?package=coinbase-pro-node-api

**Comment by alexfernandez** [comment]
It's working for me right now, maybe it was busy indexing packages (which is the usual problem I suspect).

**Comment by jcubic** [comment]
There is issue again http://npm.packagequality.com/shield/jquery.terminal.svg give `{"error":"database not available"}`

**Comment by alexfernandez** [comment]
Thanks for letting me know! I have restarted it, and scheduled a daily restart so it doesn't get stuck any more :blush:

**Comment by Shuunen** [comment]
dead on my side

**Comment by alexfernandez** [comment]
Sorry, server was out of space. Thanks!
